dags/longitudinal.py

```diff
@@ -40,6 +40,17 @@
     uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/addon_recommender.sh",
     dag=dag)
 
+ensemble_recommender = EMRSparkOperator(
+    task_id="ensemble_recommender",
+    job_name="Extract the addon data for each Telemetry user to DynamoDB",
+    execution_timeout=timedelta(hours=10),
+    instance_count=20,
```

Comment:
Just want to verify that this timeout and instance count make sense for this job -- does it require 20 nodes?